.github/workflows/deploy-changed-cf.yaml

```diff
@@ -33,13 +33,19 @@ jobs:
         name: Check CloudFormation templates
         run: |-
           # Merge all modified and renamed files into one JSON array
-          #jq -s add ${HOME}/files_added.json ${HOME}/files_modified.json ${HOME}/files_renamed.json > ${HOME}/files_to_check.json
+          jq -s add ${HOME}/files_added.json ${HOME}/files_modified.json ${HOME}/files_renamed.json > ${HOME}/files_to_check.json
 
-          cat ${HOME}/files.json
+          echo "files_to_check.json"
+          cat ${HOME}/files_to_check.json
           echo
+          #Select those that are CF templates (path starts with `cf/`)
+          jq '[.[] | select(match("^cf/"))]' ${HOME}/files_to_check.json > ${HOME}/cf_files.json
 
-          # Select those that are CF templates (path starts with `cf/`)
-          jq '[.[] | select(match("^cf/"))]' ${HOME}/files.json > ${HOME}/cf_files.json
+          echo "files.json"
+          cat ${HOME}/files.json
+          echo
+          # # Select those that are CF templates (path starts with `cf/`)
+          # jq '[.[] | select(match("^cf/"))]' ${HOME}/files.json > ${HOME}/cf_files.json
```

Comment:
This is failing if I switch back to using files.json (I added and later removed a file, but now this file always shows up in files.json so the workflow fails).

With files_to_check.json I get some duplicate files (same file in files_renamed and files_added for example), so we could work on improving that logic and remove duplicates, but I need someone who might have run into this to first say if this change would make sense, or to explain why we are using files.json.